rust/src/lib.rs

```diff
@@ -14,13 +14,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#[path = "pb/apache.rocketmq.v2.rs"]
```

Comment:
@CherishCai Why include! over specifying path?
rust-analyzer works better, say jump to definition if specifying mod by path.